consts.go

```diff
@@ -14,4 +14,6 @@ const (
 	MEAL_LIST = SKHU_URL + "uni_zelkova/uni_zelkova_4_3_list.aspx"
 	// 학식 게시판 주소
 	MEAL_BOARD = SKHU_URL + "uni_zelkova/uni_zelkova_4_3_view.aspx?idx="
+	// 학사일정 주소
```

Comment:
Codacy found an issue: [comment on exported const SCHEDULES_URL should be of the form "SCHEDULES_URL ..."](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091219)